apps/base/tasks_banking.py

```diff
@@ -149,6 +154,21 @@ def load_ofx(ofx_file):
     )
 
 
+@base.cli.command("sync_wisetransfer")
+@click.argument("profile_id", type=click.INT, required=False)
+def sync_wisetransfer(profile_id):
+    """Sync transactions from all accounts associated with a Wise profile"""
+    if profile_id is None:
+        profile_id = wise_business_profile()
```

Comment:
Again, just checking my understanding: Wise returns an arbitrary (single) business profile ID from their "list profiles" endpoint.

Is there a dangerous failure mode here where a user runs this sync command without a `profile_id` argument (since it's not required) and undefined results occur?

(there is an [exception raised](https://github.com/emfcamp/Website/blob/b39577dd4e164f3299ee9325d871f26bc5589409/apps/payments/wise.py#L198) when multiple profiles are returned by the endpoint, but it's unclear to me from the [comment](https://github.com/emfcamp/Website/blob/b39577dd4e164f3299ee9325d871f26bc5589409/apps/payments/wise.py#L191) whether that can happen)

Reply:
This is safe, because `sync_wise_statement` only processes BankAccounts which are marked as active in the admin page. The command will also currently fail if we suddenly get new, unrecognised BankAccounts, but I see no reason we shouldn't just fetch all profiles. (A BankAccount corresponds to a balance in Wise terminology, which is tied to a borderless_account_id and currency. Only one BankAccount per currency will be marked as active currently.)

Reply:
Also, I expect we'll set `TRANSFERWISE_PROFILE_ID` on production, so `wise_business_profile` will always return that.

Reply:
Ok, understood.  Setting `TRANSFERWISE_PROFILE_ID` sounds important in that case, and I'm good with this :+1: